eclipse/test/resources/launch/Saros_STF_Alice.launch

```diff
@@ -16,7 +16,7 @@
 <stringAttribute key="org.eclipse.jdt.launching.JRE_CONTAINER" value="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 <stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="-os ${target.os} -ws ${target.ws} -arch ${target.arch} -nl en_US"/>
 <stringAttribute key="org.eclipse.jdt.launching.SOURCE_PATH_PROVIDER" value="org.eclipse.pde.ui.workbenchClasspathProvider"/>
-<stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-Xmx512m -ea -XX:MaxPermSize=256m -Djava.security.manager &quot;-Djava.security.policy=file:${workspace_loc:saros.eclipse}/test/resources/stf/all.policy&quot;  -Dsaros.debug=true -Dsaros.testmode=12345 -Dsaros.sleepTime=200 -Djava.rmi.server.hostname=localhost -Dorg.eclipse.swtbot.keyboard.strategy=org.eclipse.swtbot.swt.finder.keyboard.MockKeyboardStrategy  -Dorg.eclipse.swtbot.keyboard.layout=saros.stf.server.bot.default -Duser.language=en -Duser.country=US -Dfile.encoding=UTF-8 -Djava.util.Arrays.useLegacyMergeSort=true"/>
+<stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-Xmx512m -ea -XX:MaxPermSize=256m -Djava.security.manager &quot;-Djava.security.policy=file:${workspace_loc:saros.eclipse}/test/resources/all.policy&quot;  -Dsaros.debug=true -Dsaros.testmode=12345 -Dsaros.sleepTime=200 -Djava.rmi.server.hostname=localhost -Dorg.eclipse.swtbot.keyboard.strategy=org.eclipse.swtbot.swt.finder.keyboard.MockKeyboardStrategy  -Dorg.eclipse.swtbot.keyboard.layout=saros.stf.server.bot.default -Duser.language=en -Duser.country=US -Dfile.encoding=UTF-8 -Djava.util.Arrays.useLegacyMergeSort=true"/>
```

Comment:
Regarding @m273d15's stacktrace: The relative path to the policy file should probably be changed, too.

Reply:
I fixed the configs and now I was able to run an STF-Test. @m273d15 please try again.